app/ledger.js

```diff
@@ -441,9 +448,24 @@ var initialize = (onoff) => {
         }
 
         getStateInfo(state)
+
         try {
-          client = ledgerClient(state.personaId,
-                                              underscore.extend(state.options, { roundtrip: roundtrip }, clientOptions), state)
+          client = ledgerClient(
```

Comment:
except that this is the only example of that style of indentation in the file...

perhaps we can compromise by putting the first parameter right after the "(", and the ")" right after the final parameter?